catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplicationConfig.java

```diff
@@ -36,82 +36,98 @@
 import org.openmetadata.catalog.slackChat.SlackChatConfiguration;
 
 public class CatalogApplicationConfig extends Configuration {
-  @JsonProperty("database")
+  private static final String DATABASE_CONFIG = "database";
+  private static final String SWAGGER_CONFIG = "swagger";
+  private static final String AUTHORIZER_CONFIG = "authorizerConfiguration";
+  private static final String AUTHENTICATION_CONFIG = "authenticationConfiguration";
+  private static final String JWT_CONFIG = "jwtTokenConfiguration";
+  private static final String ELASTIC_CONFIG = "elasticsearch";
+  private static final String EVENTHANDLER_CONFIG = "eventHandlerConfiguration";
+  private static final String AIRFLOW_CONFIG = "airflowConfiguration";
+  private static final String SLACK_CONFIG = "slackEventPublishers";
+  private static final String FERNET_CONFIG = "fernetConfiguration";
+  private static final String HEALTH_CONFIG = "health";
+  private static final String MIGRATION_CONFIG = "migrationConfiguration";
+  private static final String SANDBOXMODE_CONFIG = "sandboxModeEnabled";
+  private static final String SLACK_CHAT_CONFIG = "slackChat";
+  private static final String SECRET_MANAGER_CONFIG = "secretsManagerConfiguration";
+
```

Comment:
@mohitdeuex and @vivekratnavel, no need to define private static variables. We can just use the string as it is used only once and with in this class.